src/features/culturalSurvey/helpers/__tests__/utils.test.ts

```diff
@@ -1,14 +1,98 @@
 import { mockedCulturalSurveyQuestions } from 'features/culturalSurvey/fixtures/mockedCulturalSurveyQuestions'
 import { createInitialQuestionsList } from 'features/culturalSurvey/helpers/utils'
 
+import {
+  CulturalSurveyQuestionEnum,
+  CulturalSurveyAnswerEnum,
+  CulturalSurveyQuestion,
+} from './../../../../api/gen/api'
+import { getSubQuestionsToDisplay } from './../utils'
+
+function getCurrentQuestion(question: CulturalSurveyQuestionEnum) {
+  return mockedCulturalSurveyQuestions.questions.find(
+    (questionItem) => questionItem.id === question
+  ) as CulturalSurveyQuestion
+}
+
 describe('createInitialQuestionsList', () => {
   it('should return list of questions id', () => {
     const ids = createInitialQuestionsList(mockedCulturalSurveyQuestions)
-    expect(ids).toEqual(['SORTIES', 'ACTIVITES'])
+    expect(ids).toEqual([CulturalSurveyQuestionEnum.SORTIES, CulturalSurveyQuestionEnum.ACTIVITES])
   })
 
   it('should return empty list if no data provided', () => {
     const ids = createInitialQuestionsList()
     expect(ids).toEqual([])
   })
 })
+
+describe('getSubQuestionsToDisplay', () => {
+  it('should be empty', () => {
```

Comment:
"should be empty if..." ? Tu peux ajouter dans la description du `it` le cas pour lequel tu souhaites que ça soit vide, ça aidera à la compréhension ultérieure du test. Ici je pense que c'est quelque chose du genre 'it should be empty if selected answers have no subquestions' ?